Tasks/AzureRmWebAppDeployment/CompressionUtility.ps1

```diff
@@ -0,0 +1,48 @@
+﻿function UnzipWebDeployPkg
```

Comment:
You can simply use the .Net classes to do the unzip.

.Net 4.5 is already a demand of the 1.x Windows agent. We are adding prereq validation in the 2.x Windows agent.